app/src/components/sections/ExpenseAccountView/ApprovedExpensesSection.vue

```diff
@@ -141,11 +142,13 @@ const approveUser = async (data: BudgetLimit) => {
     signature: signature.value
   }
   await executeAddExpenseData()
-  reload.value = true
+  // reload.value = true
+  expenseStore.setReload(true)
```

Comment:
can you remove the comments
```suggestion
  expenseStore.setReload(true)
```